src/bin/miri.rs

```diff
@@ -374,6 +374,9 @@ fn main() {
                     miri_config.tag_raw = true;
                     miri_config.check_number_validity = true;
                 }
+                "-Zmiri-drop-stdout-stderr" => {
```

Comment:
"drop" in Rust has a particular meaning, so maybe a different verb is better? I was thinking of "silence", "silent", or "quiet".

Also please add this to the README.